homeassistant/components/switch/xiaomi_aqara.py

```diff
@@ -137,7 +137,7 @@ def parse_data(self, data, raw_data):
             self._load_power = round(float(data[LOAD_POWER]), 2)
 
         value = data.get(self._data_key)
-        if value is None:
+        if value is None or value == 'unknown':
```

Comment:
Will this work and be more future proof?
`        if value not in ['on', 'off']`

Reply:
Good idea, thanks